deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -12,5 +12,6 @@
 {{- $excludePodRegex := .Values.fluentd.logs.containers.excludePodRegex | quote }}
 {{- $excludeContainerRegex := .Values.fluentd.logs.containers.excludeContainerRegex | quote }}
 {{- $excludeHostRegex := .Values.fluentd.logs.containers.excludeHostRegex | quote }}
+{{- $clusterName := include "sumologic.clusterNameReplaceSpaceWithDash" . }}
 
-{{- $yamlFile | replace "processors.source.collector.replace" $_collector | replace "exporters.zipkin.url_replace" $endpoint | replace "processors.source.name.replace" $sourceName | replace "processors.source.category.replace" $sourceCategory | replace "processors.source.category_prefix.replace" $sourceCategoryPrefix | replace "processors.source.category_replace_dash.replace" $sourceCategoryReplaceDash | replace "processors.source.exclude_namespace_regex.replace" $excludeNamespaceRegex | replace "processors.source.exclude_pod_regex.replace" $excludePodRegex | replace "processors.source.exclude_container_regex.replace" $excludeContainerRegex | replace "processors.source.exclude_host_regex.replace" $excludeHostRegex | nindent 2 }}
\ No newline at end of file
+{{- $yamlFile | replace "processors.source.collector.replace" $_collector | replace "exporters.zipkin.url_replace" $endpoint | replace "processors.source.name.replace" $sourceName | replace "processors.source.category.replace" $sourceCategory | replace "processors.source.category_prefix.replace" $sourceCategoryPrefix | replace "processors.source.category_replace_dash.replace" $sourceCategoryReplaceDash | replace "processors.source.exclude_namespace_regex.replace" $excludeNamespaceRegex | replace "processors.source.exclude_pod_regex.replace" $excludePodRegex | replace "processors.source.exclude_container_regex.replace" $excludeContainerRegex | replace "processors.source.exclude_host_regex.replace" $excludeHostRegex | replace "processors.resource.cluster.replace" $clusterName | nindent 2 }}
```

Comment:
nit: split it on several lines to make reviews easier in the future

Reply:
@sumo-drosiek totally agree, tried to split into several lines but must have been doing it wrong and the chart never rendered; do you know the right way to do it?

Reply:
I believe something like that:
```
{{- $yamlFile = replace "processors.source.collector.replace" $_collector $yamlFIle }}
```